doc/ref_arch/openstack/chapters/chapter03.md

```diff
@@ -307,8 +309,58 @@ Most cloud storage architectures incorporate a number of clustered storage nodes
+-	controller nodes setup in-line with the vendor recommendation (e.g. min 3 physical nodes)
+-	network nodes (where applicable) 
+-	storage nodes setup in line with the vendor recommendation
+-	compute nodes – enough to handle the workload after local failure scenario
+
+<a name="3.5.1"></a>
+### 3.5.1. Topology Overview
+
+Notes:
+- Region is represented by a single OpenStack control plane.
+- Resource Failure Domain is effectively the “blast radius” of any major infrastructure failure such as loss of PDU or network leafs.
+- Control planes includes redundant network nodes where OVS-kernel is used.
+- Controller nodes are typically a minimum of 3.
+- Shared storage is optional but it is important to ensure shared assets are distributed across serving clouds such as boot images.
+
+| Topology Ref| Type| Control Planes| Shared Storage (optional)| Compute AZs| Achievable Service Availability %| Service Multi-region awareness| Notes |
+|----------|-----|-----------------|---------------|-----------|---------|--------------|-------------|
+| 1 | Local Redundancy - workload spread across servers | 1 | 1 | 1 | Variable | Not required | Suitable where only limited local application availability is required e.g. nova anti-affinity |
+| 2 | Regional Redundancy - workload spread across AZs | 1 | >=2 | >=2 | >99.n | Not required | Suitable where local application HA is required. Control plane should be distributed across DC failure domains (assuming layer 2 connectivity) but may be unavailable during upgrades |
+| 3 | Global Redundancy - workload spread across multiple Regions | >=2 | >=2 | >=2 | >99.nn | Required e.g. GSLB | Suitable where local and region application HA is required Control plane could be kept available in one site during upgrades |
+
+<a name="3.5.2"></a>
+### 3.5.2. Topology Detail 
+
+#### 3.5.2.1. Topology 1	- Local Redundancy
+
+Under normal operation this deployment can handle a single failure of a controller node or storage node without any impact to the service.   If a compute node fails the application layer (often the VNFm) would need to restart workloads on a spare compute node of similar capability i.e. cloud may need to be provided with n+1 capacity.  In the case of an active/active application deployed to separate compute nodes (with hypervisor anti-affinity) then there would be no service impact.  
+
+*Important to consider:*
+
+-	Where possible servers should be distributed and cabled to reduce the impact of any failure e.g. PDU, rack failure.   Because each operator has individual site constraints this document will not propose a standard rack layout.
+-	During maintenance of the control plane the API may not be available and for some applications this could cause problems when relying on it during BAU application operation and additionally if the upgrade involves updating OpenStack services on the compute nodes.  OVS-kernel operations may also be impacted during this time.
```

Comment:
Only the control plane APIs should be affected and so shouldn't impact running workloads

Reply:
You can also mention that while control plane is impacted, the data plane (or forwarding plane) is unimpacted

Reply:
> Only the control plane APIs should be affected and so shouldn't impact running workloads

Agree it shouldn't affect running workloads but if VNFm has to do a scale it will break.

Reply:
I am making this change now - @pgoyal01 @sukhdevkapur please check it later today.